### PR TITLE
Admin Page: remove extra space before comma.

### LINKED ISSUE #4455: SupportCard: spacing issue within text
Found a space between a comma and "twitter" that needs to be removed:

screen shot 2016-07-20 at 12 18 09 pm

### PR DESCRIPTION
**Pull request by jeherve**
Fixes #4455

### CODE REVIEW

**Comment by eliorivero** [comment]
Space before comma gone
follow
and is still ok on mobile
mobile
However, shouldn't `twitter` and `facebook` be capitalized since they are brands?

**Comment by jeherve** [comment]
Good call. e3dc998800d0b6a19ac4285242e71cf95633b61a

**Comment by eliorivero** [comment]
LGTM 🚀 
Moving to 4.3-public-beta since it can be merged after tests pass.

File: _inc/client/components/jetpack-notices/index.jsx [hunk 1]
```diff
@@ -154,7 +154,7 @@ export const DevModeNotice = React.createClass( {
 				devModeType += __('your site URL lacking a dot (e.g. http://localhost).');
 			}
 
-			const text = __('Currently in {{a}}Development Mode{{/a}} VIA ' + devModeType + '{{br/}}Some features are disabled.',
+			const text = __('Currently in {{a}}Development Mode{{/a}} Via ' + devModeType + '{{br/}}Some features are disabled.',
```
Comment (zinigor) [inline _inc/client/components/jetpack-notices/index.jsx]: I know that this PR was not meant to fix this, but let's do this anyway. We should use the full text every time we call the translation function, there should be no concatenation here. Even if that means that `Currently in...' part will be repeated several times.

**Comment by zinigor** [comment]
Read, found one problem, otherwise everything LGTM!

File: _inc/client/components/jetpack-notices/index.jsx (hunk 1 above)
Comment (jeherve) [inline _inc/client/components/jetpack-notices/index.jsx]: I'm not entirely sure what you had in mind. Would this work for you? 859b70987a01ad65e4ab3fb93827182d3c3c5d07

That's a lot of repetition though.

**Comment by eliorivero** [comment]
Needs a rebase.

**Comment by oskosk** [comment]
LGTM !

**Pull request merged**